src/Compilers/VisualBasic/Test/Emit/Attributes/AttributeTests_WellKnownAttributes.vb

```diff
@@ -3763,15 +3821,86 @@ end class
 
             Dim comp = CompilationUtils.CreateCompilationWithMscorlib40AndVBRuntime(source)
             comp.VerifyDiagnostics(Diagnostic(ERRID.ERR_OmittedArgument2, "FileIOPermission").WithArguments("action", "Public Overloads Sub New(action As System.Security.Permissions.SecurityAction)"),
```

Comment:
>VerifyDiagnostics [](http://example.com/codeflow?start=17&length=17)

Please use AssertTheseDiagnostics instead. Otherwise, we cannot tell how the error message is going to look like.

Reply:
I used it for new tests, but makes sense to use in this modified existing test too. Will do.